frontend/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:1.3
 FROM --platform=$BUILDPLATFORM node:18-alpine as dev
 
-ARG APP_SHELL_VERSION=next
+ARG APP_SHELL_VERSION=4.4.1
```

Comment:
This should be 4.5.0, like the rest of the esm-core files

Reply:
Thanks for the catch. This should look good now... :-)